base/Run-profile-image.md

```diff
@@ -0,0 +1,44 @@
+## Running the IBM WebSphere Application Server Classic Base Profile image
+
+When the container is started using the IBM WebSphere Application Server Classic Base Profile image, it executes the start script
+which takes the following environment variables
+
+* UPDATE_HOSTNAME(optional, has to be provided if hostname has to be changed from localhost)
```

Comment:
need to indicate this is a boolean e.g. "set to `true` if the hostname should be updated from the default of `localhost`"